.circleci/config.base.yml

```diff
@@ -569,6 +569,7 @@ workflows:
             branches:
               only:
                 - master
+                - layers-rework
```

Comment:
Do we need to run these tests on layers

Reply:
Whoops, this is a mistake. Meant to exclude these from the nightly tests, good catch!